packages/terra-form-select/src/_Frame.module.scss

```diff
@@ -30,55 +30,55 @@
     }
 
     &.is-focused {
-      background-image: var(--terra-select-focus-background-image, none);
+      background-image: var(--terra-form-select-focus-background-image, none);
       background-position: 0 100%;
       background-repeat: no-repeat;
-      background-size: var(--terra-select-focus-background-size, 0 0);
-      border-color: var(--terra-select-focus-border-color, #4cb2e9);
-      box-shadow: var(--terra-select-focus-box-shadow, 0 0 0 2px rgba(76, 178, 233, 0.25));
+      background-size: var(--terra-form-select-focus-background-size, 0 0);
+      border-color: var(--terra-form-select-focus-border-color, #4cb2e9);
+      box-shadow: var(--terra-form-select-focus-box-shadow, 0 0 0 2px rgba(76, 178, 233, 0.25));
       outline: none;
-      transition-duration: var(--terra-select-focus-transition-duration, 0s);
+      transition-duration: var(--terra-form-select-focus-transition-duration, 0s);
       transition-property: background-size;
-      transition-timing-function: var(--terra-select-focus-transition-timing-function, ease);
+      transition-timing-function: var(--terra-form-select-focus-transition-timing-function, ease);
     }
 
     &.is-open {
       border-bottom-left-radius: 0;
       border-bottom-right-radius: 0;
-      border-top-left-radius: var(--terra-select-border-radius, 3px);
-      border-top-right-radius: var(--terra-select-border-radius, 3px);
+      border-top-left-radius: var(--terra-form-select-border-top-left-radius, 3px);
+      border-top-right-radius: var(--terra-form-select-border-top-right-radius, 3px);
       box-shadow: none;
     }
 
     &.is-above {
-      border-bottom-left-radius: var(--terra-select-border-radius, 3px);
-      border-bottom-right-radius: var(--terra-select-border-radius, 3px);
+      border-bottom-left-radius: var(--terra-form-select-border-bottom-left-radius, 3px);
```

Comment:
nit: I'd incoporate the is-above in the variable name. e.g. --terra-form-select-above-border-top-left-radius.

Same for the bottom border: --terra-form-select-above-border-top-right-radius.

This allows us the ability to use --terra-form-select-border-top-left-radius and --terra-form-select-border-top-right-radius with different values under the .select class if we ever need to in the future.

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/08c864fe81968bcf218d28099650960c2f45b51d#diff-a7694d0174d676146b494561fb4ffb04R54

Reply:
Resolved in this commit: https://github.com/cerner/terra-core/pull/2125/commits/08c864fe81968bcf218d28099650960c2f45b51d